src/app/static/src/app/store/load/actions.ts

```diff
@@ -90,9 +92,17 @@ export const actions: ActionTree<LoadState, RootState> & LoadActions = {
         //File hashes have now been set for session in backend so we save the state from the file we're loading into local
         //storage then reload the page, to follow exactly the same fetch and reload procedure as session page refresh
         //NB load state is not included in the saved state so we will default back to NotLoading on page reload.
+
+        const {modelCalibrate} = savedState
+        if (modelCalibrate && Object.keys(modelCalibrate.options).length === 0) {
```

Comment:
I think we'd better put a comment on here to explan why we're doing this. Also, could chcek `modelCalibrate.result` to only do this for projects which have already calibrated (any new calibrates should get their options set since you fixed the bug). `modelCalibrate` itself will always be set, so that check isn't really doing anything at the moment.

  ```suggestion
       // Backwards compatibility fix: projects which calibrated before bug fix in mrc-3126 have empty calibrate options
        const {modelCalibrate} = savedState
        if (modelCalibrate.result && Object.keys(modelCalibrate.options).length === 0) {
```